domains/transfer-agreements/TransferAgreements.API/API/Metrics/ITransferAgreementAutomationMetrics.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace API.Metrics
+{
+    public interface ITransferAgreementAutomationMetrics
+    {
+        void SetNumberOfTransferAgreementsOnLastRun(int numberOfTransferAgreementsOnLastRun);
+        void SetCertificatesTransferredOnLastRun(int certificatesTransferredOnLastRun);
+        void TransferRetry(Guid certificateId);
```

Comment:
Suggest renaming this to "AddTransferRetry"

Reply:
AddTransferAttempt